addons/account/models/company.py

```diff
@@ -278,6 +280,73 @@ def _validate_fiscalyear_lock(self, values):
             if nb_draft_entries:
                 raise ValidationError(_('There are still unposted entries in the period you want to lock. You should either post or delete them.'))
 
+    def _install_localization_packages(self, from_init=False):
+        for company in self:
```

Comment:
Do not do the for-loop.  We are only going to install one company at a time.  Better to use ensure_one

Reply:
@jco-odoo because we also call from write so I think it's good to do in for loop

Reply:
ok, I put for loop in write method. 👍